ncm-ceph/src/main/perl/Ceph/daemon.pm

```diff
@@ -208,6 +208,11 @@ sub check_empty {
             return 0;
         }
     } else {
+        my $checkmntcmd = ['/usr/bin/findmnt', $loc ];
```

Comment:
@stdweird util-linux. I thought it was by default included in OS, but apparently we include it in a template for all our servers:)